README.md

```diff
@@ -200,26 +200,33 @@ object featuring a `CaptureError.CAPTURE_NO_MEDIA_FILES` error code.
 
 ### iOS Quirks
 
-Since iOS 10 it's mandatory to add a `NSCameraUsageDescription`, `NSMicrophoneUsageDescription` and `NSPhotoLibraryUsageDescriptionentry` in the info.plist.
+Since iOS 10 it's mandatory to add a `NSCameraUsageDescription`, `NSMicrophoneUsageDescription` and `NSPhotoLibraryUsageDescriptionentry` in the info.plist. If you don't add them, the app will crash when it tries to get the permission to use them. Also, Apple will reject apps that use this features and don't provide the usage descriptions.
 
-* `NSCameraUsageDescription` describes the reason that the app accesses the user’s camera.
-* `NSMicrophoneUsageDescription` describes the reason that the app accesses the user’s microphone.
+* `NSCameraUsageDescription` describes the reason that the app accesses the user's camera.
+* `NSMicrophoneUsageDescription` describes the reason that the app accesses the user's microphone.
 * `NSPhotoLibraryUsageDescriptionentry` describes the reason the app accesses the user's photo library.
 
 When the system prompts the user to allow access, this string is displayed as part of the dialog box.
 
-To add this entry you can pass the following variables on plugin install.
+To add this entries you can use `edit-config` tag in the config.xml like this:
```

Comment:
this => these

Reply:
good catch, I copy/pasted from the media PR and missed some changes 🙈